data/data-pipeline/data_pipeline/etl/score/etl_utils.py

```diff
@@ -108,3 +113,167 @@ def floor_series(series: pd.Series, number_of_decimals: int) -> pd.Series:
     )
 
     return floored_series
+
+
+def _create_df_from_yaml_contents(
```

Comment:
I put this in `etl_utils` because that seemed like the best place for it. Happy to move it though, if you disagree!

Reply:
Seems like a good home!

Reply:
Would definitely be good to test this function.